bundles/org.openhab.core.transform/src/main/java/org/openhab/core/transform/TransformationRegistry.java

```diff
@@ -21,16 +21,16 @@
 import org.openhab.core.common.registry.Registry;
 
 /**
- * The {@link TransformationConfigurationRegistry} is the interface for the transformation configuration registry
+ * The {@link TransformationRegistry} is the interface for the transformation registry
  *
  * @author Jan N. Klug - Initial contribution
  */
 @NonNullByDefault
-public interface TransformationConfigurationRegistry extends Registry<TransformationConfiguration, String> {
+public interface TransformationRegistry extends Registry<TransformationConfiguration, String> {
```

Comment:
The `TransformationRegistry` should (by the definition of its name) be a registry for `Transformations`, not `TransformationConfigurations`. Does anything speak against refactoring `TransformationConfiguration` to `Transformation`? I know that we have already the action with that name, but as this is in a different (and internal) package, this should hopefully not block us.

Reply:
I agree. My idea was to keep it backward compatible, but with renaming the `TransformationConfigurationRegistry` to `TransformationRegistry` we need to adapt the transformation add-ons that use it in openhab-addons anyway, so I'll change it.